incuna-sass/_normalize.sass

```diff
@@ -44,7 +44,7 @@ html
 // Links
 
 // Remove the gray background color from active links in IE 10.
-@if support-legacy-browser("ie", "10")
+.lt-ie11 &
```

Comment:
`As of Internet Explorer 10, conditional comments are no longer supported by standards mode. `
https://msdn.microsoft.com/en-us/library/ms537512(v=vs.85).aspx